### PR TITLE
Updating authors and hero image for blog article: https://web.dev/autowebperf/

### PR DESCRIPTION
**Pull request by jonchenn**
Changes proposed in this pull request:

- Added authors and hero image to the article https://web.dev/autowebperf/
- Added author metadata for antoinebisch

### CODE REVIEW

File: src/site/content/en/blog/autowebperf/index.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ description: >
   A new modular tool that enables automatic gathering of performance data form multiple sources.
 date: 2020-12-09
 # updated: 2020-07-22
+authors:
+  - jonchen
+  - gilbertococchi
+  - antoinebisch
+hero: hero.png
+thumbnail: hero.png
```
Comment (kaycebasques) [inline src/site/content/en/blog/autowebperf/index.md:14]:
```suggestion
```
There doesn't appear to be a thumbnail in this PR. Did you meant to add one?

File: src/site/_data/authorsData.json [hunk 2]
```diff
@@ -1680,5 +1680,19 @@
...
+      "family": "Bisch"
+    },
+    "github": "antoinebisch",
+    "country": "UK",
+    "org": {
+      "name": "Google"
+    },
+    "descriptions": {
+      "en": "Web Ecosystem Consultant"
+    }
```
Comment (kaycebasques) [inline src/site/_data/authorsData.json:1696]: We need author images for Antoine:

image

**Review by kaycebasques — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/autowebperf/index.md [hunk 3]
```diff
@@ -6,6 +6,12 @@ description: >
   A new modular tool that enables automatic gathering of performance data form multiple sources.
 date: 2020-12-09
 # updated: 2020-07-22
+authors:
+  - jonchen
+  - gilbertococchi
+  - antoinebisch
+hero: hero.png
```
Comment (kaycebasques) [inline src/site/content/en/blog/autowebperf/index.md:13]: Hero image needs to be 3200px wide by 960px tall

**Review by kaycebasques — DISMISSED** [review]

File: src/site/content/en/blog/autowebperf/index.md (hunk 1 above)
Comment (jonchenn) [reply src/site/content/en/blog/autowebperf/index.md:14]: Yes. Should I upload another low-res image for thumbnail?

File: src/site/_data/authorsData.json (hunk 2 above)
Comment (jonchenn) [reply src/site/_data/authorsData.json:1696]: Uploaded with another commit.

File: src/site/content/en/blog/autowebperf/index.md (hunk 3 above)
Comment (jonchenn) [reply src/site/content/en/blog/autowebperf/index.md:13]: Updated with another commit.

File: src/site/content/en/blog/autowebperf/index.md (hunk 1 above)
Comment (kaycebasques) [reply src/site/content/en/blog/autowebperf/index.md]: Here is updated guidance on how to use `thumbnail`: https://deploy-preview-4359--web-dev-staging.netlify.app/handbook/markup-media/#thumbnail

File: src/site/content/en/blog/autowebperf/index.md [hunk 4]
```diff
@@ -6,6 +6,11 @@ description: >
...
```
Comment (mihajlija) [inline src/site/content/en/blog/autowebperf/index.md:13]: @jonchenn Please optimize the hero image, I suggest switching to JPG. Check out our guidelines: https://web.dev/handbook/markup-media/#optimization

Comment (jonchenn) [reply src/site/content/en/blog/autowebperf/index.md]: Update images with JPEGs.

**Review by mihajlija — APPROVED** [review]

**Comment by mihajlija** [comment]
@kaycebasques This is reviewed, it just needs your approval to get merged.

**Pull request merged**